src/party.c

```diff
@@ -41,69 +41,69 @@ void InitPartyWithMaxSize(PARTY *party, int maxSize) {
     MI_CpuClear8(&party->extra, 5 * party->core.maxCount);
 }
 
-BOOL AddMonToParty(PARTY *party, const Pokemon *mon) {
+BOOL AddMonToParty(Party *party, const Pokemon *mon) {
     if (party->core.curCount >= party->core.maxCount) {
         return FALSE;
     }
     party->core.mons[party->core.curCount] = *mon;
-    MI_CpuClear8(&party->extra.unk_00[party->core.curCount], sizeof(PARTY_EXTRA_SUB));
+    MI_CpuClear8(&party->extra.unk_00[party->core.curCount], sizeof(PartyExtraSub));
     party->core.curCount++;
     return TRUE;
 }
 
-BOOL RemoveMonFromParty(PARTY *party, int slot) {
+BOOL RemoveMonFromParty(Party *party, int slot) {
     PARTY_ASSERT_SLOT(party, slot);
     GF_ASSERT(party->core.curCount > 0);
     for (; slot < party->core.curCount - 1; slot++) {
         party->core.mons[slot] = party->core.mons[slot + 1];
         party->extra.unk_00[slot] = party->extra.unk_00[slot + 1];
     }
     ZeroMonData(&party->core.mons[slot]);
-    MI_CpuClear8(&party->extra.unk_00[slot], sizeof(PARTY_EXTRA_SUB));
+    MI_CpuClear8(&party->extra.unk_00[slot], sizeof(PartyExtraSub));
     party->core.curCount--;
     return TRUE;
 }
 
-int GetPartyMaxCount(const PARTY *party) {
+int GetPartyMaxCount(const Party *party) {
```

Comment:
Party_GetMaxCount etc (add prefix for all function that take primary Party * argument in this file)